pattern-library/fonts/edx-icons/edx-icons.svg

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Cool, it may make sense to add a canned comment in all files that are generated by IcoMoon (.svg, .css, etc.)